src/main/include/subsystems/Drivetrain.h

```diff
@@ -52,7 +52,7 @@
  * Subsystem responsible for driving the robot chassis, and housing all the logic to control the
  * 4 swerve modules on the robot.
  */
-class Drivetrain : public ValorSubsystem
```

Comment:
Same thing as before, can we add the InitSendable function declaration to ValorSubsystem and make it pure virtual, so that all subsystems have to implement it?